pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgadmin_types.go

```diff
@@ -21,6 +21,8 @@ import (
 
 // PGAdminConfiguration represents pgAdmin configuration files.
 type PGAdminConfiguration struct {
+	Files []corev1.VolumeProjection `json:"files,omitempty"`
```

Comment:
:wrench: I think we'd want a comment here, perhaps something like:
"Files allows the user to mount projected volumes into the pgAdmin container, so that files can be referenced by pgAdmin as needed."